JASP-Engine/JASP/R/correlation.R

```diff
@@ -1099,37 +1001,38 @@ Correlation <- function(dataset=NULL, options, perform="run", callback=function(
 
 .createNonparametricConfidenceIntervals <- function(x, y, hypothesis = "two-sided", confLevel = 0.95, method = "kendall"){
   alpha <- 1 - confLevel
+  missingIndices <- as.logical((is.na(x)) + (is.na(y)))
+  x <- x[!missingIndices]
+  y <- y[!missingIndices]
   n <- length(x)
-  corValue <- cor.test(x, y, method=method)$estimate
-  if(method == "kendall" && sum(is.infinite(c(x,y)))==0){
+  
+  corValue <- cor.test(x, y, method = method)$estimate
+  if ( method == "kendall" && sum(is.infinite(c(x,y))) == 0 ) {
```

Comment:
What happens if one of the entries is infinite? Moreover, isn't this checked by .hasErrors already?